WooCommerce/Classes/Notifications/PushNotificationsManager.swift

```diff
@@ -480,16 +480,16 @@ private extension PushNotificationsManager {
 
 // MARK: - ForegroundNotification Extension
```

Comment:
nit: this maybe also be updated to `PushNotification`?

Reply:
🤦  Fixed in a499f5856fce27fa87ce56fd2edb87af1b58a54d